components/script/dom/webidls/Window.webidl

```diff
@@ -201,3 +201,10 @@ partial interface Window {
    readonly attribute TestRunner testRunner;
    //readonly attribute EventSender eventSender;
 };
+
+// https://drafts.css-houdini.org/worklets/#examples
+partial interface Window {
+   [Pref="dom.worklet.testing.enabled", SameObject] readonly attribute Worklet testWorklet;
+   [Pref="dom.worklet.testing.enabled"] DOMString? testWorkletLookup(DOMString key);
```

Comment:
What if we made a separate DOM interface with a constructor so we don't need to include test-only fields in Window?